src/ui/uploader.js

```diff
@@ -64,15 +64,40 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         }
     });
 
-    /* Updates the uploader's internal file representation with
-     * information stored in the DOM's uploader element.
+    /* Updates the uploader's internal file representation with information
+     * stored in the DOM's uploader element. If the file is an image, an attempt
```

Comment:
The comment here says "an attempt will be made to rotate images", which doesn't seem to happen in this function. Do I miss anything?

Reply:
indeed, the call to `loadImage` is the attempt to rotate images (as you suggested below, I'll be providing some more info on this)